DQM/EcalCommon/data/writeEcalDQMStatus.py

```diff
@@ -15,10 +15,10 @@
 process.EcalTrivialConditionRetriever.producedEcalDQMChannelStatus = False
 process.EcalTrivialConditionRetriever.producedEcalDQMTowerStatus = False
 
-process.load("CondCore.DBCommon.CondDBCommon_cfi")
-process.CondDBCommon.DBParameters.authenticationPath = '/nfshome0/popcondev/conddb'
-#process.CondDBCommon.connect = 'sqlite_file:mask-ECAL.db'
-#process.CondDBCommon.connect = 'oracle://cms_orcon_prod/CMS_COND_34X_ECAL'
+process.load("CondCore.CondDB.CondDB_cfi")
+process.CondDB.DBParameters.authenticationPath = '/nfshome0/popcondev/conddb'
+process.CondDB.connect = 'sqlite_file:mask-ECAL.db'
```

Comment:
@thomreis This is the change producing a valid sqlite... nothing to do with conddb dump

Reply:
Without changing the `writeOne()` functions in DQM/EcalMonitorDbModule/plugins/EcalDQMStatusWriter.cc from pointer based to reference based the produced sqlite file was not valid. Or at least the payload in it could not be printed.